plugin/pkg/admission/antiaffinity/admission.go

```diff
@@ -59,7 +60,8 @@ func (p *plugin) Admit(attributes admission.Attributes) (err error) {
 	}
 	affinity, err := api.GetAffinityFromPodAnnotations(pod.Annotations)
 	if err != nil {
-		return err
+		glog.Warningf("Invalid NodeAffinity detected, but we will leave handling of this to validation phase")
```

Comment:
We generally wouldn't output this at WARNING because on a large cluster there could be thousands of these and they would fill the logs.  You can rely on validation being called for the pod, because if it isn't, then lots of other very scary things can happen in the cluster.

Putting it at glog.V(4) or 5 is ok.